containers/src/Kokkos_DualView.hpp

```diff
@@ -552,6 +591,41 @@ template< class , class , class , class > friend class DualView ;
 #endif
   }
 
+  inline void modify_host() const {
```

Comment:
The methods are defined inline in their class, so there's no need to mark them `inline`.